src/util/fetchUtils.js

```diff
@@ -94,8 +98,17 @@ export const mergeCookies = (rawCookieHeader, newCookies) => {
 		...oldCookies,
 		...newCookies,
 	};
-	return Object.keys(mergedCookies)
-		.map(name => `${name}=${mergedCookies[name]}`)
-		.join('; ');
+	return stringifyCookies(mergedCookies);
+};
+
+export const BAD_COOKIES = [
```

Comment:
this might be the simplest way to deal with this but it would be nice if we didn't need to blacklist.